### ISSUE TITLE
A JSON-RPC API is not available(omni_listblocktransactions)

### ISSUE THREAD

**Issue by qshuai** [OmniLayer/omnicore#963]
`omni_listblocktransactions firstblock lastblock`

```
omnicore-cli --datadir=.omnicore/ omni_listblocktransactions 581092 581093
error code: -1
error message:
omni_listblocktransactions index

Lists all Omni transactions in a block.

Arguments:
1. index                (number, required) the block height or block index

Result:
[                       (array of string)
  "hash",                 (string) the hash of the transaction
  ...
]

Examples:
> omnicore-cli omni_listblocktransactions 279007
> curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "omni_listblocktransactions", "params": [279007] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/
```

**Comment by dexX7** [comment]
Hi, there is a bit of confusion due to a typo in the help and RPC descriptions. There are two RPCs:

- omni_listblocktransactions index
- omni_listblockstransactions firstblock lastblock

The first one can be used to list the transactions of a single block, while the second one can be used to list transactions of multiple blocks.

It's fixed with #957.

**Comment by qshuai** [comment]
@dexX7 Thanks, got it.
